lib/utils/new-module.js

```diff
@@ -153,11 +153,17 @@ function getFullNames(prefix, node) {
   return fullNames;
 }
 
-function isInitImportedFrom(node, module) {
+function isDestructuring(node) {
+  return node.type === 'VariableDeclarator' && node.init.type === 'Identifier';
```

Comment:
There are helpers in `utils/utils.js` that we should use for checking types.

Reply:
I won’t be behind a computer for the coming  10 days. If we have to merge it before my return, I have to leave this change to you. I’m sorry for that 😬.